service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -97,6 +97,13 @@ public int updateLastRunWithRunSet(RunSet runSet) {
                 "method_id", runSet.methodVersion().method().methodId())));
   }
 
+  public int unsetLastRunSetId(UUID methodId) {
+    String sql =
+        "UPDATE method SET %s = NULL WHERE %s = :method_id"
+            .formatted(Method.LAST_RUN_SET_ID_COL, Method.METHOD_ID_COL);
+    return jdbcTemplate.update(sql, new MapSqlParameterSource(Map.of("method_id", methodId)));
+  }
```

Comment:
I'm adding these new DAO methods because "updateLastRunWithRunSet" is (apparently) not nullable. I could have tried to get around that, but having a separate explicit DAO method for unsetting this seemed reasonable to me.

Reply:
This seems sensible. It kind of makes sense that this is a separate activity from updating it